Sources/BraintreeThreeDSecure/Public/BraintreeThreeDSecure/BTThreeDSecureRequest.h

```diff
@@ -20,16 +20,30 @@
 NS_ASSUME_NONNULL_BEGIN
 
 /**
- Error codes that describe errors that occur during 3D Secure.
+ 3D Secure version
  */
-typedef NS_ENUM(NSInteger, BTThreeDSecureVersion){
+typedef NS_ENUM(NSInteger, BTThreeDSecureVersion) {
     /// 3DS 1.0
     BTThreeDSecureVersion1,
 
     /// 3DS 2.0
     BTThreeDSecureVersion2
 };
 
+/**
+ The account type
+ */
+typedef NS_ENUM(NSInteger, BTThreeDSecureAccountType) {
+    /// Unspecified
+    BTThreeDSecureAccountTypeUnspecified,
```

Comment:
FYI - the only reason we need an "unspecified" option is because Objective-C enums cannot be nil.